### PR TITLE
[3.x] Add link to edit method for table columns.

### PR DESCRIPTION
**Pull request by haringsrob**
Adds a ->linkToEdit method to table builder columns

### CODE REVIEW

File: docs/src/crud-modules/tables.md [hunk 1]
```diff
@@ -64,6 +64,7 @@ All columns are placed under the `src/Services/Listings/Columns` directory.
 - `sortable(bool $sortable = true)`: Makes a column sortable.
 - `optional(bool $optional = true)`: Makes a column optional.
 - `hide(bool $hide = true)`: Hide the field by default. Only usable with optional.
+- `linkToEdit(bool $linkToEdit = true)`: Link the filed to the edit form.
```
Comment (ifox) [inline docs/src/crud-modules/tables.md:67]:
```suggestion
- `linkToEdit(bool $linkToEdit = true)`: Link the field to the edit form.
```

File: ide.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (ifox) [inline ide.json:1]: Should this be git ignored? If not, could you add usage docs to the contributing section?

File: src/Http/Controllers/Admin/ModuleController.php [hunk 3]
```diff
@@ -1024,7 +1034,7 @@ public function show($id, $submoduleId = null)
      * @param int|null $submoduleId
      * @return \Illuminate\Http\JsonResponse|\Illuminate\Http\RedirectResponse|\Illuminate\View\View
      */
-    public function edit(TwillModelContract | int $id, $submoduleId = null)
+    public function edit(TwillModelContract|int $id, $submoduleId = null)
```
Comment (ifox) [inline src/Http/Controllers/Admin/ModuleController.php:1037]: Was the code style ruleset updated for those stylistic changes to happen on this PR?

**Review by ifox — CHANGES_REQUESTED** [review]

File: ide.json (hunk 2 above)
Comment (haringsrob) [reply ide.json:1]: This was intentional.

It does not do anything outside of phpstorm, but it is a staring idea and currently provides autocomplete for TwillNavigation:

Screenshot 2022-09-26 at 12 34 31

File: src/Http/Controllers/Admin/ModuleController.php (hunk 3 above)
Comment (haringsrob) [reply src/Http/Controllers/Admin/ModuleController.php:1057]: The previous formatting was incorrect, however, there's no specific rules changed.

File: ide.json (hunk 2 above)
Comment (ifox) [reply ide.json:1]: Very nice! Thanks for clarifying.

File: src/Http/Controllers/Admin/ModuleController.php (hunk 3 above)
Comment (ifox) [reply src/Http/Controllers/Admin/ModuleController.php:1057]: Sounds good!

**Review by ifox — APPROVED** [review]

**Pull request merged**